docs/routing-rules.md

```diff
@@ -134,6 +134,21 @@ condition: 'request.getHeader("X-Trino-Client-Tags") contains "label=foo"'
 
 If no rules match, then request is routed to adhoc.
 
+### TrinoStatus
```

Comment:
Please fix broken style: 
![Screenshot 2024-10-08 at 8 23 35](https://github.com/user-attachments/assets/3781c718-15f1-4eda-af96-0bdbef6867c4)

Reply:
it should be fixed now. It looks like 
![image](https://github.com/user-attachments/assets/c2971efd-185a-4a98-8019-07dc1f9f8526)
on my IDE's preview

Reply:
Ideally test with mkdocs locally.

Reply:
yep, tested it 
![image](https://github.com/user-attachments/assets/d1769e0c-6f04-4795-b67a-b549821bd4e0)